tsc cannot find declaration module for request after upgrading from 6.4.0 to 6.5.0

#### Issue Summary

After dependabot upgraded our library to v6.5.0 from v6.4.0, we ran into this error on CI:

```
../../node_modules/@sendgrid/client/src/request.d.ts:1:30 - error TS7016: Could not find a declaration file for module 'request'. '/home/circleci/project/node_modules/request/index.js' implicitly has an 'any' type.
  Try `npm install @types/request` if it exists or add a new declaration (.d.ts) file containing `declare module 'request';`

1 import {OptionsWithUrl} from "request";
                               ~~~~~~~~~
```

It appears to be a missing dependency. `@types/request` is marked as a devDependency, but the `.d.ts` makes it a non-dev dependency, if that makes sense.

#### Technical details:

* sendgrid-nodejs Version: 6.5.0
* Node.js Version: 12.14.0

--- a/packages/client/package.json
+++ b/packages/client/package.json
@@ -28,10 +28,8 @@
   },
   "dependencies": {
     "@sendgrid/helpers": "^6.5.1",
+    "@types/request": "^2.48.4",
     "request": "^2.88.0"
-  },
-  "devDependencies": {
-    "@types/request": "^2.0.3"
   },
   "tags": [
     "http",

